### ISSUE TITLE
Improve navigation of Webinars wiki

### ISSUE THREAD

**Issue by tucotuco** [VertNet/dwc-qa-manage#17]
I keep finding that I want to go to the iDigBio announcement for a webinar and there is no link for it on the webinars page. It might be good to review navigation on this page (https://github.com/tdwg/dwc-qa/wiki/Webinars) in general.

**Comment by ekrimmel** [comment]
+1

**Comment by pzermoglio** [comment]
@tucotuco I see what you mean, but I'm not sure that the announcements are that important to people once the webinar is over...? They are useful to us, yes, though.
Would the following do..:
- List all announcements in the manage repo Wiki.
- List announcements of webinars that have not yet come to pass in the Q&A repo in place of the video link until the video is up to replace it.
If all agree, I'd be happy to implement the change.

**Comment by ekrimmel** [comment]
@pzermoglio I find myself needing the iDigBio announcement to check the time of upcoming webinars, and to see who presented past webinars or is presenting upcoming webinars. We could solve both of these needs by doing what you suggest, plus adding the presenters' names to the abstracts on the Q&Q wiki.

**Comment by tucotuco** [comment]
Agreed to add announcement column to contain title still. Change content of presentations column.

**Comment by debpaul** [comment]
The "relevant links" information - shared for each webinar, is another resource we gain when linking to the announcements. I'll start this now change now and see how it goes. We can tweak the format. I can't quite remember what we said for how the presentation (slides) column should look.

**Comment by debpaul** [comment]
@tucotuco @pzermoglio @ekrimmel @garymotz @dagendresen @hollyel 
- [x] Deb added column to the https://github.com/tdwg/dwc-qa/wiki/Webinars table.
- [ ] To be done: add presenter names, - where? with the abstracts?, in a column?

**Comment by tucotuco** [comment]
I like what you have done with the columns.  To me it seems reasonable to put the presenters in the area below with the abstracts.

**Comment by debpaul** [comment]
@tucotuco @pzermoglio @ekrimmel @garymotz @dagendresen @hollyel anything else need tweaking on this webinar page navigation? For example @tucotuco, you suggest that perhaps we put the presenters' names with the abstracts?
